modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -752,6 +760,7 @@ public boolean equals(Object o) {
         return isAlias == that.isAlias &&
                 isString == that.isString &&
                 isInteger == that.isInteger &&
+                isShortInteger == that.isShortInteger &&
```

Comment:
Suggestion: What about just `isShort` as we've `isLong` (instead of `isLongInteger`)?

Reply:
This has been updated to isShort